scripts/atomizer-loop.py

```
@@ -0,0 +1,113 @@
+import subprocess
+
+def sql2(connection, command, data):
+    attempts = 0
+    while attempts < 3:
+        try:
+            cursor = connection.cursor(dictionary=True)
+            cursor.execute(command, data)
+            if "UPDATE" in command or "INSERT" in command:
+                connection.commit()
+            return [x for x in cursor]
+        except Exception as e:
```

Ruff (0.11.9)

Local variable `e` is assigned to but never used

Remove assignment to unused variable `e`

(F841)